src/components/common/imager/Imager.tsx

```diff
@@ -76,6 +77,18 @@ export const Imager = ({
   );
 };
 
+export const TokenLogo = ({ token, size }: { token: Token; size: number }) => {
```

Comment:
I think we need to ensure that logos are always a circle. Probably worth using `LogoImager`